### PR TITLE
Alexmcsw profile update

### PR DESCRIPTION
**Pull request by alexmcsw**

### CODE REVIEW

**Comment by ianabc** [comment]
Hi Alex,

I think your files are in the wrong place. Can you try editing them to move them? They should be in a folder called `content/authors`, e.g.

```
content/authors/alexmcsw/_index.md
```

Other than that I think this looks good. When you choose edit file it should let you change the path to the file but if you can't find it let me know.

File: content/authors/alexmcsw/_index.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+- Category Theory
+- Representation Theory
+- Women's History of Math
+
+# Your educational history, add as many as you want
+education:
+  courses:
+  - course: Honours Bachelor of Science in Mathematics
+    institution: University of Ottawa
+    - course: Master's of Science in Mathematics
```
Comment (ccontrer) [inline content/authors/alexmcsw/_index.md:31]: Hi Alex,

This line appears to be the issue. Can you try removing the space before `- course` to have both at the same indentation level? 

Is the syntax indent-sensitive @ianabc ?

Comment (ccontrer) [reply content/authors/alexmcsw/_index.md:31]: It looks like indentation **is** important in YAML.

Comment (alexmcsw) [reply content/authors/alexmcsw/_index.md]: Fixed, thanks!

**Comment by ianabc** [comment]
Looks great, thank you! Merging now.

**Pull request merged**